Doctrine/EventSubscriber.php

```diff
@@ -64,7 +66,12 @@ public function persistObject(DataEvent $event)
         }
 
         $objectManager->persist($event->getData());
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
on success too, this executes whatever happens in the `try` block. The code after the try/finally block is executed only if the flow of execution has not been broken, for example by a non catch exception. You can also use `finally` with `catch`, it works fine